src/app/components/publiclists/teamlist/teamcard/teamcard.component.html

```diff
@@ -1,9 +1,6 @@
 <div class="team-card-container" [class.selected]='isSelected' [class.noexpand] = 'isHost' [class.expand] = '!isHost'>
 <div class="team-card-container-child">
     <div class="team-content" (click)='selectToggle()'>
-      <input type="radio" class="with-gap selectPhase" (click)='selectTeamToggle()' *ngIf='!isSelected && (isHost || isOnChallengePage)' id="{{teamView['team_name']}}">
-      <input type="radio" class="with-gap selectPhase" (click)='selectTeamToggle()' aria-hidden="true" *ngIf='isSelected && (isHost || isOnChallengePage)' id="{{teamView['team_name']}}">
-      <label for="{{teamView['team_name']}}"></label>
```

Comment:
@Suryansh5545 Why have you removed both inputs?

Reply:
i think i misunderstood what you said before sir .

Reply:
if i am right i only have to remove the condition?

Reply:
right, but if you remove the condition do you think you need another input there? so you can remove that extra input also.

Reply:
i have done the changes Sir please have a look.

Reply:
@Sanji515 Sir the extra input is there because the id of the radio button shown is dynamic which will make both of them checkable.